simplyblock_web/templates/storage_init_job.yaml.j2

```diff
@@ -0,0 +1,54 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: snode-spdk-deployment-{{ HOSTNAME }}
+  namespace: {{ NAMESPACE }}
+spec:
+  template:
+    spec:
+      restartPolicy: OnFailure
+      nodeSelector:
+        kubernetes.io/hostname: {{ HOSTNAME }}
+      hostNetwork: true
+      hostPID: true
+      serviceAccountName: simplyblock-storage-node-sa
+      tolerations:
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+      containers:
+        - name: init-setup
+          image: ubuntu:22.04
+          securityContext:
+            privileged: true
+          command: ["/bin/bash", "-c"]
+          args:
+            - |
+              set -e
+
+              echo "--- Installing curl & networking tools ---"
+              apt-get update && apt-get install -y iproute2 curl grep
+
+              NODE_IP=$(ip route get 1.1.1.1 | grep -oE 'src [0-9.]+' | awk '{print $2}')
+              echo "Detected node IP: $NODE_IP"
+
+              echo "--- Sending config to $NODE_IP ---"
+              RESPONSE=$(curl -s -o /dev/null -w "%{http_code}" -X POST http://$NODE_IP:5000/snode/apply_config -H "Content-Type: application/json" -d '{}')
```

Comment:
does `snode/apply_config` also applies the huge pages configuration? 

```
sysctl -w vm.nr_hugepages=$hugepages
```

if so since we are already doing everything. Should we ask user to configure the huge pages any more as a part of. node preparation steps? 

https://github.com/simplyblock-io/simplyblock-csi/blob/master/docs/storage-nodes.md#preparing-nodes